dist/tools/pyterm/pyterm

```diff
@@ -106,6 +106,8 @@ class SerCmd(cmd.Cmd):
         self.run_name = run_name
         self.log_dir_name = log_dir_name
         self.newline = newline
+        if formatter:
+            self.fmt_str = formatter
```

Comment:
Uärghs, what is this, my eyes!!1! ^^, please at least use [`hasattr()`](https://docs.python.org/3/library/functions.html#hasattr) ;-)